docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
:windows: :see_no_evil: 

But it's OK, we can keep it.

Reply:
well, it's generated :)